services/findAudioStart.py

```diff
@@ -0,0 +1,52 @@
+import librosa
+import numpy as np
+import sys
+
+def dtw_shift_param(sig1, sig2, sr):
+    n_fft = int((sr / 10.) * 2.)
+    hop_length = int(n_fft / 2.)
+
+    sig1_chroma = librosa.feature.chroma_stft(y=sig1, sr=sr, n_fft=n_fft, hop_length=hop_length)
+    sig2_chroma = librosa.feature.chroma_stft(y=sig2, sr=sr, n_fft=n_fft, hop_length=hop_length)
+
+    _, wp = librosa.sequence.dtw(X=sig1_chroma, Y=sig2_chroma, metric='euclidean')
+    return wp, hop_length
```

Comment:
민석님! 크로마그램(chromagram)을 계산한 뒤, DTW 알고리즘을 적용하는 방식을 파이썬을 통해 잘 구현해주신 것 같습니다! 👍
다만, 검색해보니 크로마그램과 DTW 알고리즘 자체가 메모리의 사용이 많을 수 있어 연산이 끝난 변수는 바로바로 지워줄 수 있다고 합니다!  좀 더 찾아보고 결과값도 도출해본 후 괜찮다면 적용해봐도 좋을 것 같습니다 :)

```suggestion
    sig1_chroma = librosa.feature.chroma_stft(y=sig1, sr=sr, n_fft=n_fft, hop_length=hop_length)
    sig2_chroma = librosa.feature.chroma_stft(y=sig2, sr=sr, n_fft=n_fft, hop_length=hop_length)
     
     del sig1, sig2

    _, wp = librosa.sequence.dtw(X=sig1_chroma, Y=sig2_chroma, metric='euclidean')
    return wp, hop_length
```

Reply:
네 민지님! 민지님 말씀대로 찾아보니 파이썬은 변수 메모리를 명시적으로 해제할 수 있네요! 저희 작업이 메모리를 많이 사용하는만큼 최적화할 수 있는 부분이 있다면 꼭 하는 것이 좋겠네요! 알려주셔서 감사합니다.